### PR TITLE
Edit FSM feature

### PR DESCRIPTION
**Pull request by sumboid**

### CODE REVIEW

File: plugins/fsm/src/plugin.ts [hunk 1]
```diff
@@ -11,7 +11,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import type { Ref } from '@anticrm/core'
+import isEqual from 'lodash/isEqual'
```
Comment (haiodo) [inline plugins/fsm/src/plugin.ts:14]: Could you use: fast-equals instead, it is faster and already used.

**Comment by sumboid** [comment]
In case you want to try it:
1. Join `FSM` space (temporary step)
2. Create vacancy
3. Create candidate and then assign him to the vacancy

After that you be able to to see board as part of vacancy and edit fsm, for better experience you can open vacancy from the `Card` viewlet.

File: plugins/fsm/src/plugin.ts (hunk 1 above)
Comment (sumboid) [reply plugins/fsm/src/plugin.ts]: Done

File: plugins/fsm/package.json [hunk 2]
```diff
@@ -34,9 +34,11 @@
     "eslint-plugin-promise": "^5.1.0",
     "eslint-plugin-standard": "^5.0.0",
     "eslint-plugin-svelte3": "^3.1.2",
+    "fast-equals": "^2.0.1",
+    "lodash": "^4.17.21",
```
Comment (haiodo) [inline plugins/fsm/package.json:38]: I we do not need a lodash

**Review by z2sx — APPROVED** [review]

**Pull request merged**